packages/salesforce-adapter/src/custom_object_instances_deploy.ts

```diff
@@ -23,7 +23,7 @@ import {
   TypeElement,
   SaltoElementError,
   SaltoError,
-  SeverityLevel,
+  SeverityLevel, isInstanceElement,
```

Comment:
Style comment - this is more consistent with current import mechanism
```suggestion
  SeverityLevel,
  isInstanceElement,
```